### ISSUE TITLE
BWAPILauncher error message unclear when missing config file

### ISSUE THREAD

**Issue by malcolmsharpe** [OpenBW/openbw#4]
When running BWAPILauncher without a config file, it simply prints:

    Error: file_reader: failed to open  for reading

(This message observed on MacOS and Linux.)

This is a bit confusing, because it's not clear what file it was trying to read. (In fact, I suspect this is not complaining about failing to read the config file, but about failing to read the map, which presumably has a blank default value.)

I'd propose printing an error (or at least warning) when the config file isn't found, as well as clarifying the error when failing to read the map.

**Comment by tscmoo** [comment]
You're right, it has a blank default value. An error message would be useful here.

**Comment by shedatc** [comment]
I submitted some fixes to help there.
